src/Main/Main.tsx

```diff
@@ -1,141 +1,177 @@
-import { SxProps, styled } from '@mui/material';
+import MenuIcon from '@mui/icons-material/Menu';
+import { AppBar, Stack } from '@mui/material';
+import Box from '@mui/material/Box';
+import CssBaseline from '@mui/material/CssBaseline';
+import Drawer from '@mui/material/Drawer';
+import IconButton from '@mui/material/IconButton';
+import Toolbar from '@mui/material/Toolbar';
+import { styled } from '@mui/material/styles';
 
-import React, { Component } from 'react';
+import * as React from 'react';
 
 import { Context } from '@graasp/sdk';
 
-import Header from '../Header';
-import Sidebar from '../Sidebar';
-import { DRAWER_WIDTH } from '../constants';
+import { AccentColors, PRIMARY_COLOR } from '../theme';
 
-const StyledRoot = styled('div')({
-  display: 'flex',
-  height: '100%',
-});
+const drawerWidth = 240;
 
-const DrawerHeaderContainer = styled('div')(({ theme }) => ({
-  display: 'flex',
-  alignItems: 'center',
-  padding: '0 8px',
-  ...theme.mixins.toolbar,
-  justifyContent: 'flex-end',
+const buildHeaderGradient = (color: string): string =>
+  `linear-gradient(90deg, ${PRIMARY_COLOR} 0%, ${PRIMARY_COLOR} 35%, ${color} 100%);`;
```

Comment:
It is wanted to have primary color twice (0% and 35%) ? If yes, shouldn't be possible to only use `linear-gradient(90deg, ${PRIMARY_COLOR} 35%, ${color} 100%);` ?

Reply:
I think the idea was to have a gradient that would not be symmetrical, more blue on the left side. I tried and it looks like I can get rid of the '0%' color indication. So maybe that is what I will do ! Thank you for the comment.